ublox_serial_demo/src/main.rs

```diff
@@ -1,16 +1,20 @@
-use ublox::{Device, Position};
-use std::time::Duration;
 use chrono::prelude::*;
+use std::time::Duration;
+use ublox::{Device, Position};
 
 fn main() {
```

Comment:
This is better suited as an example, perhaps `examples/serial.rs`? (or I guess, `examples/serial.rs` under the `ublox` crate)

Reply:
Done